db/migrate/20231102141929_add_index_on_event_external_subscription_id.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddIndexOnEventExternalSubscriptionId < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_index(
+      :events,
+      %i[organization_id external_subscription_id code timestamp],
+      name: 'index_events_on_external_subscription_id_and_code_and_timestamp',
+      where: '(deleted_at IS NULL)'
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)